@types/index.d.ts

```diff
@@ -82,35 +86,39 @@ interface RequestInit {
 	highWaterMark?: number;
 }
 
-interface ResponseInit {
+export interface ResponseInit {
 	headers?: HeadersInit;
 	status?: number;
 	statusText?: string;
```

Comment:
Should we add `readonly` here?

Reply:
I agree

Reply:
Well, it makes sense to use the least demanding types - in the `dom.lib` most of the field declared as `readonly`